### DISCUSSION TITLE
problem with passing verification as a student

### DISCUSSION THREAD

**Discussion by masshitro** [community/community#65879] [GitHub Education]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Good day, I am a student from Ukraine, and we are currently undergoing online education because our city was almost destroyed, and we relocated to Germany. How can I theoretically meet the requirement of being in the city or near your campus? Also, I apologize, but how can I take a selfie with a not-so-great camera that the robot would accept? I translated most of it using the system's requirements, but it was not accepted, and it said there was a lack of clarity. We believe you should be near the campus or university.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by denverharry** [comment]
I am also facing the same issue on the [yes or no button](https://yesornobutton.org/) website.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @masshitro ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
